QUEST_LV_0100.tsv

```diff
@@ -1415,100 +1415,100 @@ QUEST_LV_0100_20150317_001414	{memo X}The Barrier Device is on the left end of h
+QUEST_LV_0100_20150317_001459	$The Goddess has surely sent help!{nl}Those monsters were last seen south of here.
+QUEST_LV_0100_20150317_001460	$There are still a lot of things I must do...
 QUEST_LV_0100_20150317_001461	{memo X}The adventure can continue if I get my diary back.{nl}I am happy..
-QUEST_LV_0100_20150317_001462	Long time no see. {nl}I have a favor to ask you. Can you do me a favor?{nl}
-QUEST_LV_0100_20150317_001463	I want to put a monument up for my dead colleague.{nl}But, I don't have any materials or tools.
+QUEST_LV_0100_20150317_001462	$Long time no see. {nl}I have a favor to ask you. Can you help?{nl}
+QUEST_LV_0100_20150317_001463	$I want to put a monument up for my dead colleague.{nl}But I don't have any materials or tools.
 QUEST_LV_0100_20150317_001464	{memo X}
 QUEST_LV_0100_20150317_001465	{memo X}Tools for stonemasons? {nl}10,000 Silvers will do.{nl}{nl}
 QUEST_LV_0100_20150317_001466	$I just figured out how I'm going to make it.{nl}I will make it quick, so please wait.
 QUEST_LV_0100_20150317_001467	{memo X}
-QUEST_LV_0100_20150317_001468	I keep thinking about my colleague.{nl}I'll never see him again, right..
-QUEST_LV_0100_20150317_001469	Soul of Varkis
+QUEST_LV_0100_20150317_001468	$I keep thinking about my colleague.{nl}I'll never see him again, right..
+QUEST_LV_0100_20150317_001469	$Soul of Varkis
 QUEST_LV_0100_20150317_001470	{memo X}I can't leave with all my things left here
-QUEST_LV_0100_20150317_001471	I don't have any reason to stay in this land anymore.{nl}I will go meet the goddess..
+QUEST_LV_0100_20150317_001471	I don't have any reason to stay in this land anymore.{nl}I want to meet the goddesses...
 QUEST_LV_0100_20150317_001472	{memo X}It was made long ago, but any stones will break it.
-QUEST_LV_0100_20150317_001473	Monument
+QUEST_LV_0100_20150317_001473	$Monument
 QUEST_LV_0100_20150317_001474	$We have all become like wind in the canyon.{nl}Although we have left, our times together will always remain with you.
-QUEST_LV_0100_20150317_001475	Architect Laudi
-QUEST_LV_0100_20150317_001476	$I received a request to rebuild this structure, but after Medzio Diena, I have been stuck here.{nl}I don't know if the original client is still alive or if we lack the materials,{nl} but I will finish rebuilding this structure no matter what happens.
+QUEST_LV_0100_20150317_001475	$Architect Laudi
+QUEST_LV_0100_20150317_001476	$I received a request to rebuild this structure, but after Medzio Diena, I have been stuck here {nl}since. I don't know if the original client is still alive or if we lack the materials, but I will finish {nl}rebuilding this structure no matter what happens.
 QUEST_LV_0100_20150317_001477	{memo X}I will find the key to open the last seal.{nl}After opening a space with this scroll.
-QUEST_LV_0100_20150317_001478	The space may exist anywhere, but it may also not exist anywhere.
+QUEST_LV_0100_20150317_001478	$The space may exist anywhere, but it might also not exist anywhere.
```

Comment:
I don't think the line refers to a specific space.

And its actually for the spatial chasms so I'll change it to be more precise.